Library/Formula/agda.rb

```diff
@@ -51,38 +56,81 @@ def install
     cabal_clean_lib
 
     if build.with? "stdlib"
-      resource("stdlib").stage prefix/"agda-stdlib"
+      resource("stdlib").stage prefix/"lib"
```

Comment:
Is this not going to cause issues for other formulae/user installations that rely on the existing path?